lib/srv/server/installer/autodiscover_test.go

```diff
@@ -87,6 +89,18 @@ func setupDirsForTest(t *testing.T, testTempDir string, distroConfig map[string]
 	}
 }
 
+type mockGCPInstanceGetter struct{}
+
+// GetInstance gets a GCP VM.
```

Comment:
I think we could remove the godocs here (and below), the type is private and the methods are not implemented.